src/RetryMiddleware.php

```diff
@@ -209,7 +209,7 @@ private function bindStatsToReturn($return, array $stats)
 
     private function isThrowable($value)
     {
-        return class_exists(\Throwable::class)
+        return interface_exists('Throwable', false)
```

Comment:
Calling instanceof on a non-exitence class is fine. PHP won;t try to autoload it because it's clever enough to see that it's clearly `false` if the definition hasn't even been loaded yet.